docs/src/catalyst_functionality/constraint_equations.md

```diff
@@ -23,19 +23,21 @@ There are several ways we can create our Catalyst model with the two reactions
 and ODE for $V(t)$. One approach is to use compositional modeling, create
 separate `ReactionSystem`s and `ODESystem`s with their respective components,
 and then extend the `ReactionSystem` with the `ODESystem`. Let's begin by
-creating these two systems:
+creating these two systems. 
+
+Here, to create differentials with respect to time (for our differential equations), we must import the time differential operator from Catalyst. We do this through `import Catalyst: D_nounits as D` (calling it `D`). Here, `D(V)` denotes the differential of the variable `V` with respect to time.
```

Comment:
Should we add a similar `D = default_D` too?

Reply:
Or maybe `D = default_time_deriv()` would be better?

Reply:
I will add `D = default_time_deriv()` for now. On the fence of what to call it, but don't think it matters that much anyway (and we can change later easily).